pkg/cli/admin/release/info.go

```diff
@@ -775,6 +775,20 @@ func (o *InfoOptions) LoadReleaseInfo(image string, retrieveImages bool) (*Relea
 				errs = append(errs, err)
 				return true, nil
 			}
+			for _, tag := range is.Spec.Tags {
+				// If ForcePrefix true, use user-provided image rather than it's mirrored source
```

Comment:
nit: "it's" -> "its"

Reply:
> > To test this PR, do something like this...
> 
> But then your suggestion doesn't mention `--force-prefix`?

oops, fixed.. plus, I am going to rework this to use ICSP, but wanted to push this in the meantime bc it's simpler than the other PR.

Reply:
> nit: "it's" -> "its"
thanks, that one always gets me